tasks/setup-RedHat.yml

```diff
@@ -4,10 +4,15 @@
     name: erlang
     state: present
 
-- name: Add packagecloud GPG key.
+- name: Add GPG keys.
   rpm_key:
-    key: "https://packagecloud.io/gpg.key"
+    key: "{{ item }}"
     state: present
+  with_items:
+    - "https://packagecloud.io/gpg.key"
+    - "https://packagecloud.io/rabbitmq/erlang/gpgkey"
+    - "https://packagecloud.io/rabbitmq/rabbitmq-server/gpgkey"
+    - "https://github.com/rabbitmq/signing-keys/releases/download/2.0/rabbitmq-release-signing-key.asc"
```

Comment:
Is this relevant for future releases of rabbitmq as well, though?

Reply:
Sorry, I don't understand the question. Are you referring to some announcement about changes planned for future RabbitMQ releases?

The keys listed here match the ones that RabbitMQ currently recommends, see https://www.rabbitmq.com/install-rpm.html#package-cloud.